main_school

```diff
@@ -32,9 +30,9 @@ def exit_program
   exit!
 end
 
-def mainSchool
+def main_school
```

Comment:
- [ Required ] I think it would be better if you ensure that after refactoring your code the application works the same way as before. Currently, when I run the project I get the error below. Kindly fix this.

## Error when running the program
![Screenshot from 2023-10-24 23-14-11](https://github.com/GeekyHacks/OOPSchoolLibrary/assets/16392046/56414967-f8d2-4d6f-a578-065e7a1ad478)